docs/infrastructure/android-build-infrastructure.md

```diff
@@ -7,7 +7,10 @@ sidebar_position: 2
 
 # Android Build Infrastructure
 
-For each Android build, Appcircle creates a brand new virtual machine running Debian Linux (v. 11).;
+For each Android build, Appcircle creates a brand new virtual machine;
+
+If you select "Default Intel Pool", virtual machine will be Debian 11 Bullseye.
+If you select "Default M1 Pool", virtual machine will be macOS Ventura 13.2.1
```

Comment:
When "Default M1 Pool" is selected, we have two options, like iOS builds. But the only difference is that the end user cannot select a macOS version for Android builds.

Both `Monterey` and `Ventura` have the same capabilities for Android builds. So both of them have an equal chance to execute an Android build.

Reply:
Added info about both ventura and monterey versions. Gived a info section.